app/javascript/dashboard/routes/dashboard/conversation/search/PopOverSearch.vue

```diff
@@ -83,6 +83,9 @@ export default {
       conversations: 'conversationSearch/getConversations',
       uiFlags: 'conversationSearch/getUIFlags',
     }),
+    currentPage() {
+      return this.$store.state.conversationPage.currentPage;
```

Comment:
I guess this should be a getter defined in the state.

Reply:
There is already a getter 'conversationPage/getCurrentPage' but it requires an argument (the current assignee tab). Should I add another getter that doesn't require the argument to conversationPage.js?

Reply:
Yes, Can you add another getter that doesn't require the argument to conversationPage.js?

Reply:
@nithindavid sure. How should I name this getter without an argument then? Or should I just enhance the existing getter and check whether the argument has been provided or not?

Reply:
If you could refactor the current `getCurrentPage` function to a new more sensible name would be the best, as there is only one usage so far.

Reply:
So I should give `getCurrentPage` a new name such as `getCurrentPageFilter` or `getCurrentFilteredPage` and then add `getCurrentPage` that just returns that object without any filter applied?

Reply:
'getCurrentPageFilter' and 'getCurrentPageWithFilter' sounds good

Reply:
To make sure I got it right:
I'm going to rename `getCurrentPage` to either `getCurrentPageFilter` or `getCurrentPageWithFilter` and create a function `getCurrentPage` that returns the object without having to provide any filter as arguments?

Reply:
Yes 👍